src/haconiwa/cli.py

```
@@ -622,6 +751,150 @@ def policy_delete(
+    # Check if tmux session exists
+    import subprocess
+    try:
+        result = subprocess.run(['tmux', 'has-session', '-t', company], 
+                               capture_output=True, text=True)
+        if result.returncode != 0:
+            typer.echo(f"❌ Company session '{company}' not found", err=True)
+            typer.echo("💡 Use 'haconiwa space list' to see available sessions", err=True)
+            raise typer.Exit(1)
+    except FileNotFoundError:
+        typer.echo("❌ tmux is not installed or not found in PATH", err=True)
+        raise typer.Exit(1)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)